lib/cloudregister/registerutils.py

```diff
@@ -499,6 +508,44 @@ def get_credentials(credentials_file):
     return (username, password)
 
 
+# ----------------------------------------------------------------------------
+def set_registry_credentials(registry_fqdn, username, password):
+    """Set the auth token to pull images from SUSE registry."""
+    auth_token = base64.b64encode('{username}:{password}'.format(
+        username=username,
+        password=password
+    ).encode()).decode()
```

Comment:
it is not because of security, it is because we _have to_, docker|podman dont like anything that is not base64

```bash
> cat ${XDG_RUNTIME_DIR}/containers/auth.json
{
	"auths": {
		"registry.suse-local.com:5000": {
			"auth": "SCC_foo:password_bar"
		}
	}
}
> podman login  --tls-verify=false
Error: get credentials: 1 error occurred:
	* illegal base64 data at input byte 72
> cat ~/.docker/config.json
{
        "auths": {
                "registry.suse-local.com:5000": {
                        "auth": "SCC_foo:password_bat"
                }
        }
}
> docker login registry.suse-local.com:5000
WARNING: Error loading config file: ~/.docker/config.json: illegal base64 data at input byte 3
Username:
```

from the [podman doc](https://docs.podman.io/en/stable/markdown/podman-login.1.html)
```
[...] Podman creates ${XDG_RUNTIME_DIR}/containers/auth.json (if the file does not exist) and then stores the username and password from STDIN as a base64 encoded string in it
```
from the [docker doc](https://docs.docker.com/reference/cli/docker/login/) 
```
 [...]
 If none of these binaries are present, it stores the credentials (i.e. password) in base64 encoding in the config files described above
 ```